jaeger-spark-dependencies-test/src/main/java/io/jaegertracing/spark/dependencies/test/DependenciesTest.java

```diff
@@ -161,6 +169,46 @@ public void testZipkinMultipleTraces() throws Exception {
     assertDependencies(expectedDependencies);
   }
 
+  @Test
+  public void testMultipleReferences() throws Exception {
+    Tuple<Tracer, Flushable> s1Tuple = TracersGenerator.createJaeger("S1", collectorUrl);
+    Tuple<Tracer, Flushable> s2Tuple = TracersGenerator.createJaeger("S2", collectorUrl);
+    Tuple<Tracer, Flushable> s3Tuple = TracersGenerator.createJaeger("S3", collectorUrl);
+
+    Span s1Span = s1Tuple.getA().buildSpan("foo")
+        .ignoreActiveSpan()
+        .start();
+    Span s2Span = s2Tuple.getA().buildSpan("bar")
+        .addReference(References.FOLLOWS_FROM, s1Span.context())
+        .start();
+    Span s3Span = s3Tuple.getA().buildSpan("baz")
+        .addReference(References.FOLLOWS_FROM, s1Span.context())
```

Comment:
Possibly change one of these to be a CHILD_OF - not that it should make a difference but just to reflect some realworld situations?

Reply:
+1, btw the reference can be of arbitrary type, there is no difference in this job.